lib/generators.nix

```diff
@@ -243,7 +243,7 @@ rec {
   toYAML = {}@args: toJSON args;
```

Comment:
Isn't this noop, too?

Reply:
could be changed to but I don't know if it's necessary
```suggestion
  toYAML = {}: toJSON {};
```